climada/test/test_blackmarble.py

```diff
@@ -199,7 +198,7 @@ def test_set_country_pass(self):
 # Execute Tests
 if __name__ == "__main__":
     TESTS = unittest.TestLoader().loadTestsFromTestCase(Test2013)
-    TESTS.addTests(unittest.TestLoader().loadTestsFromTestCase(Test1968))
-    TESTS.addTests(unittest.TestLoader().loadTestsFromTestCase(Test2012))
-    TESTS.addTests(unittest.TestLoader().loadTestsFromTestCase(BMFuncs))
+    # TESTS.addTests(unittest.TestLoader().loadTestsFromTestCase(Test1968))
+    # TESTS.addTests(unittest.TestLoader().loadTestsFromTestCase(Test2012))
+    # TESTS.addTests(unittest.TestLoader().loadTestsFromTestCase(BMFuncs))
```

Comment:
Why did you comment this out?

Reply:
uncommented now, was trying failing test and did no want to run them all.. forgot about that

Reply:
You can run specific tests using a command line like this: `python -m unittest climada.test.test_blackmarble.BMFuncs`